nuxeo-drive-jsf/src/main/resources/OSGI-INF/l10n/messages_en_US.properties

```diff
@@ -22,3 +22,12 @@ user.center.nuxeoDrive.platform.ubuntu.docLinkTitle=Read the documentation about
 rootRegistered=Drive synchronization enabled
 rootUnregistered=Drive synchronization disabled
 view.drive.title.metadata=Properties
+
+# Nuxeo Drive event names
+aboutToRegisterRoot=About To Register Root
+aboutToUnRegisterRoot=About To Unregister Root
```

Comment:
- "About to disable Drive synchronization" is better
- Typo in the key, should be ""aboutToUnregisterRoot"